GUI/Utils/DarkModeForms/FlatTabControl.cs

```diff
@@ -0,0 +1,177 @@
+using GUI;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace DarkModeForms
+{
+    public partial class FlatTabControl : TabControl
+    {
+        #region Public Properties
+
+        [Description("Color for a decorative line"), Category("Appearance")]
+        public Color LineColor { get; set; } = MainForm.DarkModeCS.OScolors.Accent;
+
+        [Description("Color for all Borders"), Category("Appearance")]
+        public Color BorderColor { get; set; } = MainForm.DarkModeCS.OScolors.TextInactive;
+
+        [Description("Back color for selected Tab"), Category("Appearance")]
+        public Color SelectTabColor { get; set; } = MainForm.DarkModeCS.OScolors.Surface;
+
+        [Description("Fore Color for Selected Tab"), Category("Appearance")]
+        public Color SelectedForeColor { get; set; } = MainForm.DarkModeCS.OScolors.TextActive;
+
+        [Description("Back Color for un-selected tabs"), Category("Appearance")]
+        public Color TabColor { get; set; } = MainForm.DarkModeCS.OScolors.Control;
+
+        [Description("Background color for the whole control"), Category("Appearance"), Browsable(true)]
+        public override Color BackColor { get; set; } = MainForm.DarkModeCS.OScolors.Control;
+
+        [Description("Fore Color for all Texts"), Category("Appearance")]
+        public override Color ForeColor { get; set; } = MainForm.DarkModeCS.OScolors.TextInactive;
+
+        #endregion
+
+        public FlatTabControl()
```

Comment:
```suggestion
        public FlatTabControl() : base()
```

And remove the empty try catches